guide/glossary/index.md

```diff
@@ -201,7 +201,7 @@ A standard for multi-signature that uses Schnorr signatures. Previously, the mor
 
 ### [Node]({{ '/guide/glossary/node/' | relative_url }})
 
-Node refers to [software]({{ '/guide/getting-started/software/#nodes' | relative_url }}) that participates in the Bitcoin network. It exchanges transaction data with other nodes, stores some or all of it, and verifies that transactions are valid. There is also dedicated [node hardware]({{ '/guide/getting-started/hardware/#hardware-wallets' | relative_url }}).
+Node refers to [software]({{ '/guide/getting-started/software/#nodes' | relative_url }}) that participates in the Bitcoin network. It exchanges transaction data with other nodes, stores some or all of it, and verifies that transactions are valid. There is also dedicated [node hardware]({{ '/guide/getting-started/hardware/#nodes' | relative_url }}).
```

Comment:
Perhaps https://bitcoin.design/guide/glossary/node/#hardware-node is more suitable to the context?
```suggestion
Node refers to [software]({{ '/guide/glossary/node/#hardware-node' | relative_url }}) that participates in the Bitcoin network. It exchanges transaction data with other nodes, stores some or all of it, and verifies that transactions are valid. There is also dedicated [node hardware]({{ '/guide/getting-started/hardware/#nodes' | relative_url }}).
```

Reply:
Yeah that is better, thanks Pav !